app/src/main/java/com/example/myapplication/SecondFragment.java

```diff
@@ -100,7 +100,7 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         binding.intakeBackground.setTranslationY(height * 0.321f);
         ArrayAdapter<String> chainAdapter = new ArrayAdapter<>(view.getContext(), android.R.layout.simple_spinner_dropdown_item, new String[]{"No Intake", "Ground Intake", "Source Intake", "Both"});
         binding.intakeMethod.setAdapter(chainAdapter);
-        binding.intakeMethod.setSelection(SecondFragment2.chainAttemptIndex);
+        binding.intakeMethod.setSelection(0);
```

Comment:
This will clear the selection if the user went back from a subsequent page. This needs to pick up the data from the current match data.